packages/webpack/nx-webpack-plugin.ts

```diff
@@ -0,0 +1,2 @@
+export { NxWebpackPlugin } from './src/plugins/nx-webpack-plugin/nx-webpack-plugin';
```

Comment:
Can we tighten up the path a bit to `@nx/webpack/webpack-plugin`?

Should look like:
```
import { NxWebpackPlugin } from '@nx/webpack/webpack-plugin';
```

What does this plugin actually do? Is there a more descriptive name for this?

Reply:
The deep import path is weird because we already have `@nx/webpack/plugin` reserved for inferring targets. The `NxWebpackPlugin` handles building both web and node apps. Could be `NxAppWebpackPlugin` or whatever. It replaces what the executor used to do.